src/melee.cpp

```diff
@@ -1204,14 +1206,20 @@ bool player::can_weapon_block() const
 {
     return (weapon.has_technique( WBLOCK_1 ) ||
             weapon.has_technique( WBLOCK_2 ) ||
-            weapon.has_technique( WBLOCK_3 ));
+            weapon.has_technique( WBLOCK_3 ) ||
+            weapon.has_technique( WBLOCK_4 ) ||
+            weapon.has_technique( WBLOCK_5 ));
```

Comment:
My apologies for having not been able to attempt it until now. Replacement seems to require the function being moved below `blocking_ability` to compile correctly, compiling to test.